torch/optim/lr_scheduler.py

```diff
@@ -234,6 +247,10 @@ def get_lr(self):
         return [base_lr * lmbda(self.last_epoch)
                 for lmbda, base_lr in zip(self.lr_lambdas, self.base_lrs)]
 
+    @property
+    def steps_on(self):
+        return StepsOnFlag.EPOCH
```

Comment:
LambdaLR, and MutliplicativeLR depend on what the user wants. My impression is that we shouldn't make a choice here. The options are see here are: (1) have a way for the user to make the choice at construction, (2) let the NotImplementedError be raised, but have a way for the user to override this. Thoughts?

Reply:
see also [comment](https://github.com/pytorch/pytorch/issues/37768#issuecomment-659536888)

Reply:
The motivation for this is described [here](https://github.com/pytorch/pytorch/issues/37768#issuecomment-659365012) and [here](https://github.com/pytorch/pytorch/issues/37768#issuecomment-659564127). Namely, the decision seems to come down to whether standard/default behavior should be provided. To ensure backwards compatibility (given this likely will not land in a major release), I'd argue it should and that it should follow existing documentation, even if the described behavior is not enforced.

It certainly seems reasonable to allow the user to override this flag (upon construction, via inheritance, etc.), however, I wanted to minimize the scope of the functionality added in this PR. Given that both an extra `__init__` param or redefined behavior via inheritance would both be backwards compatible with this change (and the current API), I didn't feel particularly compelled to include it here. Happy to fold it in though if you'd like.